l10n_it_corrispettivi_fatturapa_out/views/account_invoice_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2019 Simone Rubino
+  Copyright 2019 Giovanni - GSLabIt
+  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+  -->
+
+<odoo>
+    <record id="corrispettivi_fatturapa_out_form" model="ir.ui.view">
+        <field name="name">Integrate corrispettivi and fatturapa out form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="l10n_it_fatturapa_out.view_invoice_form_fatturapa"/>
```

Comment:
Inheriting from the view of `l10n_it_corrispettivi` is crucial because it is a primary view, so editing (using inheritance) that view doesn't change the invoice views.

In fact, I have just checked that now you are hiding the button from invoices' forms too.

Reply:
will double check as it seems to be working on my local ..

Reply:
You are right, I forgot the proof :)
This is from the runbot of this PR
![image](https://user-images.githubusercontent.com/46226165/52813736-bddd3b00-309a-11e9-9ae8-8812b9a3a0d3.png)

Reply:
You are right, sorry ..i'll revert inherit to `l10n_it_corrispettivi`. Problem in doing this is i get back to: https://github.com/OCA/l10n-italy/issues/994